core-services/pr-reminder/_config.yaml

```diff
@@ -12,13 +12,15 @@ teams:
   - dmistry
   - pruan
   - lxia
+  - hvidosil
   teamNames:
   - test-platform
   repos:
   - openshift/ci-tools
   - openshift/ci-docs
   - openshift/release
   - kubernetes/test-infra
+  - kubernetes-sigs/prow
```

Comment:
what does this? how is this useful?

Reply:
it defines the repos that our team receives pr-reminder digests for. Prow moved awhile back, and this wasn't updated.

Reply:
so if I am asked to review in prow now, will I get notified?

Reply:
yes